docs/index.rst

```diff
@@ -0,0 +1,1032 @@
+                    "population":50,
+                    "standard_deviation":0.08521,
+                    "sum":6.3584,
+                    "sum_squares":1.16432,
+                    "variance":0.00726
+                },
+                "id":"000001",
+                "name":"Cluster 1"
+            }],
+            "fields":{
+
+                ...snip...
+
+            }
+        },
+        "code":200,
+        "columns":5,
+        "created":"2016-02-17T08:26:12.583000",
+        "credits":0.017581939697265625,
+        "credits_per_prediction":0.0,
+        "critical_value":5,
+        "dataset":"dataset/56c42ea07e0a8d6cca01519b",
+        "dataset_field_types":{
+            "categorical":1,
+            "datetime":0,
+            "effective_fields":5,
+            "items":0,
+            "numeric":4,
+            "preferred":5,
+            "text":0,
+            "total":5
+        },
+        "dataset_status":true,
+        "dataset_type":0,
+        "description":"",
+        "dev":true,
+        "excluded_fields":[],
+        "field_scales":{},
+        "fields_meta":{
+            "count":5,
+            "limit":1000,
+            "offset":0,
+            "query_total":5,
+            "total":5
+        },
+        "input_fields":[
+            "000000",
+            "000001",
+            "000002",
+            "000003",
+            "000004"
+        ],
+        "k":2,
+        "locale":"en_US",
+        "max_columns":5,
+        "max_rows":150,
+        "model_clusters":false,
+        "name":"Iris Source dataset's cluster",
+        "number_of_batchcentroids":0,
+        "number_of_centroids":0,
+        "number_of_public_centroids":0,
+        "out_of_bag":false,
+        "price":0.0,
+        "private":true,
+        "project":null,
+        "range":[
+            1,
+            150
+        ],
+        "replacement":false,
+        "resource":"cluster/56c42ea47e0a8d6cca0151a0",
+        "rows":150,
+        "sample_rate":1.0,
+        "scales":{
+            "000000":0.18941532079904913,
+            "000001":0.35975000221609077,
+            "000002":0.08884141152890178,
+            "000003":0.20571391803576422,
+            "000004":0.15627934742019414
+        },
+        "shared":false,
+        "size":4609,
+        "source":"source/56c42e9f8a318f66df007548",
+        "source_status":true,
+        "status":{
+            "code":5,
+            "elapsed":1213,
+            "message":"The cluster has been created",
+            "progress":1.0
+        },
+        "subscription":false,
+        "summary_fields":[],
+        "tags":[],
+        "updated":"2016-02-17T08:26:24.259000",
+        "white_box":false
+    }
+
+(Note that we have abbreviated the output in the snippet above for
+readability: the full predictive cluster you'll get is going to contain
+much more details).
+
+Anomaly Detector
+----------------
+
+For anomaly detection problems, BigML anomaly detector uses iforest as an
```

Comment:
"For anomaly detection problems," leftover? sounds strange both phrases as they are now.
"For anomaly detection problems BigML uses iforest ..." or "BigML anomaly detector uses iforest ..."